Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md

```diff
@@ -66,15 +67,19 @@ There is a specific documentation for load balancing with [Azure Web Apps](azure
 
 ## Examine Directory Factory Options
  
-- The `TempEnvDirectoryFactory` allows Examine to store indexes directly in the environment temporary storage directory, and should be used instead of `SyncTempEnvDirectoryFactory` mentioned above.
+- The `TempEnvDirectoryFactory` allows Examine to store indexes directly in the environment temporary storage directory, and should be used instead of `SyncTempEnvDirectoryFactory` mentioned below. The `TempEnvDirectoryFactory` should be used on front-end read-only servers.
```

Comment:
TempEnvDirectoryFactory can be used on back-office servers and often is as it's the option less likely to experience locking issues. I know it's not new but I also think the "should be used instead" should be removed also.

Reply:
Maybe the correct approach on this is to just clarify the differences and the benefits of each then. I'll have another look